ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -215,10 +216,13 @@ run runargs@RunNodeArgs{..} =
                         rnTraceConsensus
                         btime
                         chainDB
+                        bfsalt
       nodeKernel <- initNodeKernel nodeArgs
       rnNodeKernelHook registry nodeKernel
 
-      let ntnApps = mkNodeToNodeApps   nodeArgs nodeKernel
+
+      kaRng <- newStdGen
+      let ntnApps = mkNodeToNodeApps   nodeArgs nodeKernel kaRng
```

Comment:
Should we make the `kaRng` part of `nodeArgs`? Rename it to `keepAliveRng`.

Reply:
The tip-sample protocol also needs a StdGen.
I think we should add a StdGen to `nodeArgs` and then protocols that need pseudo random numbers can just split it.